README.md

```diff
@@ -20,7 +20,7 @@ There are a very wide range of already-written plugins available which you can u
  * [DevTools](https://github.com/pmmp/PocketMine-DevTools/) - A development tools plugin for creating plugins.
 
 ### Can I contribute?
-Yes you can! Contributions are welcomed provided that they comply with our [Contributing Guidelines](CONTRIBUTING.md). Please ensure you read the relevant sections of the guidelines carefully before making a Pull Request or opening an Issue.
+Yes you can! Contributions are welcomed provided that they comply with our [Contributing Guidelines](CONTRIBUTING.md). Please ensure you read the relevant sections of the guidelines carefully before making a Pull Request or opening an Issue. Failing to comply could result in a repository ban.
```

Comment:
Why would someone get banned for a denied contribution?

Reply:
^ I'd also like to know that.

Reply:
This looks overly aggressive... what's also funny is that this PR could result in a ban all by itself by these rules, as it doesn't comply with the contribution guidelines. 🙄 

I don't believe there's any need to be so aggressive on the README, otherwise we might well scare away potential contributors less brave than yourself.